django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -92,12 +92,7 @@
         // select (+ or :pencil:) update ForeignKey PK of the rest of selects
         // in the page.
 
-        const path = win.location.pathname;
-        // Extract the model from the popup url '.../<model>/add/' or
-        // '.../<model>/<id>/change/' depending the action (add or change).
-        const modelName = path.split('/')[path.split('/').length - (objId ? 4 : 3)];
-        // Exclude autocomplete selects.
-        const selectsRelated = document.querySelectorAll(`[data-model-ref="${modelName}"] select:not(.admin-autocomplete)`);
+        const selectsRelated = document.querySelectorAll(`[data-context="available-source"]`);
```

Comment:
I don't understand why we can drop the `data-model-ref="${modelName}"` here, what if there are multiple fields in the same form that allow for the multiple selection process?

Reply:
Hi @nessita , thanks for ur feedback. Valid point. My intention behind removing the data-model-ref="${modelName}" was to simplify the selector logic, but I understand the need for specificity to make sure we r targeting correct model. I have reintroduced the data-model-ref="${modelName}".